src/auro-accordion.js

```diff
@@ -29,6 +29,7 @@ import styleCssFixed from "./style-fixed-css.js";
  * @attr {Boolean} warning - Dependent on shade; warning styling
  * @attr {Boolean} error - Dependent on shade; error styling
  * @attr {Boolean} fixed - Uses px values instead of rem
+ * @attr {Boolean} showOverflow - Allows content to extend outside of the .details container
```

Comment:
Remove `noanimation` from here and be sure to run `npm run build` or `npm run apiBuild` to generate a new API document.

Reply:
rslvd